anax-in-container/config/anax.json

```diff
@@ -26,6 +26,8 @@
   "ArchSynonyms": {
     "x86_64": "amd64",
     "armhf": "arm",
-    "aarch64": "arm64"
+    "aarch64": "arm64",
+    "ppc64le": "ppc64le",
+    "s390x": "s390x"
```

Comment:
These 2 lines are unnecessary. You only need to list synonyms that are different from the default name of the architecture.

Reply:
Thanks, removed